test_collections/matter/sdk_tests/scripts/fetch_sdk_tests_and_runner.sh

```diff
@@ -159,28 +158,39 @@ rm -rf ${EXTRACTION_ROOT}
 
 mkdir -p ${EXTRACTION_ROOT}
 
-cd ${SDK_PATH}/scripts/py_matter_idl
-python -m build --outdir ${EXTRACTION_ROOT}
-cd ${SDK_PATH}/scripts/py_matter_yamltests
-python -m build --outdir ${EXTRACTION_ROOT}
-# Copy the chipyaml adapters and install as editable package
-cp -r ${SDK_PATH}/scripts/tests/chipyaml ${EXTRACTION_ROOT}/chipyaml
-
-# Create a simple setup.py for chipyaml in the extraction root
-cat > ${EXTRACTION_ROOT}/setup.py << 'EOF'
-from setuptools import setup, find_packages
-setup(
-    name='chipyaml',
-    version='1.0.0',
-    packages=find_packages(),
-    install_requires=[]
-)
+
+python -m build --outdir "${EXTRACTION_ROOT}" "${SDK_PATH}/scripts/py_matter_idl"
+python -m build --outdir "${EXTRACTION_ROOT}" "${SDK_PATH}/scripts/py_matter_yamltests"
+# Create chipyaml package with the full adapters structure
+mkdir -p "${EXTRACTION_ROOT}/chipyaml_src"
+cp -r ${SDK_PATH}/scripts/tests/chipyaml ${EXTRACTION_ROOT}/chipyaml_src/
+
+# Create pyproject.toml for chipyaml package at the root level
+cat > ${EXTRACTION_ROOT}/chipyaml_src/pyproject.toml << 'EOF'
+[build-system]
+requires = ['setuptools>=45', 'wheel']
+build-backend = 'setuptools.build_meta'
+
+[project]
+name = "chipyaml"
+version = "0.0.1"
```

Comment:
Having Major and Minor versions as "zero" seems strange 
```suggestion
version = "0.1.0"
```